cl/opinion_page/templates/opinion.html

```diff
@@ -403,7 +404,7 @@ <h4>Summary of Opinion</h4>
                             <a href="#{{ sub_opinion.type }}{{ forloop.counter }}"
                                aria-controls="{{ sub_opinion.type }}{{ forloop.counter }}"
                                role="tab"
-                               data-toggle="tab">{{ sub_opinion.get_type_display }}</a>
+                               data-toggle="tab">{{ sub_opinion.get_type_display|cut:"Opinion" }} {% if sub_opinion.author_str or sub_opinion.author %}<span class="text-capitalize">({% if sub_opinion.author %}{{ sub_opinion.author.name_last }}{% else %}{{ sub_opinion.author_str }}{% endif %})</span>{% endif %}</a>
```

Comment:
Can you unwind this one change and then lets merge this

Reply:
done, I'll merge it when the checks are done